nntrainer/compiler/compiler.h

```diff
@@ -7,7 +7,21 @@
  * @brief NNTrainer compiler that reads and generates executable graph
  * @details
  * Graph is convertible either to iostream, representation, executable by
- * appropriate compiler and interpreter
+ * appropriate compiler and interpreter.
+ * For example, if istream would be from a a.tflite file,
+ *
+ * GraphRepresentaion g;
+ * g.setInterpreter(tfliteInterpreter);
+ * g.setCompiler(nntrainerCPUCompiler);
+ * auto tflite_file = std::open('a.tflite');
+ * tflite_file >> g;
+ *
+ * ExecutableGraph eg = g.compile();
+ *
+ * g.setInterpreter(iniInterpreter);
```

Comment:
So.. we does not consider tflite backbone case? ( This is tflite + nntrainer case )

Reply:
This is rebased, this used to be the initial design, I just rebased :)